demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -319,11 +319,11 @@ export class DemoMeetingApp
   markdown = require('markdown-it')({ linkify: true });
   lastMessageSender: string | null = null;
   lastReceivedMessageTimestamp = 0;
-  lastReceivedPackets = 0;
+  lastReceivedPacketsSent = 0;
```

Comment:
Shouldn't this just be lastPacketsSent?

Reply:
I think either works. Basically it just means the last `packetsSent` received via `MetricsDidReceive` event. Do you have a strong intention to rename this?

Reply:
I think lastReceivedPacketsSent is misleading and could be misinterpreted as the server's packets sent instead of our packets sent